src/mappostgis.cpp

```diff
@@ -1579,6 +1579,25 @@ static int msPostGISBase64Decode(unsigned char *dest, const char *src,
 
 #endif
 
+/*
+** isPropertyNumeric()
+**
+** Check if a field is numeric
+*/
+static bool isPropertyNumeric(layerObj *layer, const char *property) {
```

Comment:
would you mind moving that into a msIsPropertyIsNumeric() method (in maplayer.c maybe?), so it can be used here, and by mapogr.cpp ?